### ISSUE TITLE
Support for Application Insights?

### ISSUE THREAD

**Issue by jellebens** [Azure/service-fabric-mesh-preview#375]
I created a custom telemetry initiualizer cfr ServiceFabric. it works in local debugging but not when i deploy to azure, any plans for this? In  local debug I do not seem to be able to tracing dependcie calls either without plumbing offcourse

In microservices this is a must (imo)

**Comment by mattrowmsft** [comment]
The SF module wouldn't work here and that wouldn't be our goal.  You should be able to build something very similar from the context passed via environment variables to each container.  We would eventually get to this if someone from the community doesn't get to it first.  App Insights folks have been driving some really cool work with OpenTelemetry that I have been keeping a close eye on as well.

**Comment by jellebens** [comment]
yes I have written my own TelemetryInitializer which effectively reads the environment variables. It works fine on my devbox but not when i publish because I assume the module or extension is not enabled there. I will try to fiddle around in the arm template a bit to get it enabled
snippit of what I get below from my local devbox
![image](https://user-images.githubusercontent.com/1615249/63342432-c82f0900-c34b-11e9-9882-074702dd8f2a.png)
